libminifi/test/unit/LoggerTests.cpp

```diff
@@ -380,7 +380,9 @@ TEST_CASE("Setting max log entry length to unlimited results in unlimited log en
 TEST_CASE("fmt formatting works with the logger") {
   LogTestController::getInstance().setTrace<logging::Logger>();
   std::shared_ptr<logging::Logger> logger = logging::LoggerFactory<logging::Logger>::getLogger();
-  logger->log_critical("{} equals to {}", 1min, 60s);
+  std::chrono::minutes duration{1};
```

Comment:
Made this update for compatibility between fmt 10.1.0 that is installed by standalone CMake approach and fmt 10.2.1 that is installed by conan approach. I verified after building MiNiFi using both approaches that the LoggerTest passes.